### ISSUE TITLE
[FEAT] New service: webex

### ISSUE THREAD

**Issue by kikanq** [tigattack/VeeamNotify#68]
### Is your feature request related to a problem?

hello, how to integrate it with webex in the company?

**Comment by tigattack** [comment]
Looking at the [Webex incoming webhook docs](https://apphub.webex.com/applications/incoming-webhooks-cisco-systems-38054-23307-75252), it seems to accept Markdown.

This will be quite simple to add support for, I'll see if I can look into it once #63 is merged since support for Webex will look very similar to that. Alternatively, if you need this quickly, you're welcome to submit your own pull request with the necessary changes.

**Comment by kikanq** [comment]
Hi !  I am not a programmer and it will be difficult for me to understand some phrases, so I will write what I managed to do after your answer.  I managed to connect weebhook to my company account - I created a room in my webex and completed all the fields in the weebhook integration, and obtained the weebhook address for my connection with webex.  Does the next step require improving the code for the weebex application?

**Comment by kikanq** [comment]
Hi !  I am not a programmer and it will be difficult for me to understand some phrases, so I will write what I managed to do after your answer.  I managed to connect weebhook to my company account - I created a room in my webex and completed all the fields in the weebhook integration, and obtained the weebhook address for my connection with webex.  Does the next step require improving the code for the weebex application?

**Comment by kikanq** [comment]
the code below when invoked locally on my laptop works (of course after entering the correct webhook ID).  the test message is visible in the weebex area I created.
----------------
Invoke-RestMethod -Uri "https://webexapis.com/v1/webhooks/incoming/YOUR_WEBHOOK_ID" -Method POST -Body '{"markdown": "Test **message**"}' -Headers @{"Content-Type"="application/json"}

**Comment by kikanq** [comment]
https://help.webex.com/en-us/article/n7i55j5/Webex-App-%7C-Markdown-formats

**Comment by tigattack** [comment]
> Does the next step require improving the code for the weebex application?

Yes that's right. It could be based heavily on the changes in #63, however I would suggest waiting until that PR is merged since the changes aren't yet complete, so you'd be basing your changes on possibly incorrect code.

**Comment by kikanq** [comment]
thanks for the information.  when do you expect the merge to be completed?

**Comment by tigattack** [comment]
Difficult to say, sorry. It's currently pending.

**Comment by kikanq** [comment]
how can I contact you privately?
